src/chainparams.h

```diff
@@ -139,13 +145,15 @@ const CChainParams &Params();
 /** Return parameters for the given network. */
 CChainParams &Params(CBaseChainParams::Network network);
```

Comment:
Of course another option is to maintain it in elements until it removed in core.